test/index.js

```diff
@@ -1,12 +1,50 @@
-import { strict as assert } from 'node:assert';
-import binCheck from 'bin-check';
-import { test } from 'uvu';
-import hugoBin from '../index.js';
+import process from "node:process";
+import binCheck from "bin-check";
+import { test } from "uvu";
+import hugoBin from "../index.js";
+import hugoLib from "../lib/index.js";
+import * as assert from "uvu/assert";
 
-test('should return path to binary and work', () => {
-  return binCheck(hugoBin, ['version']).then(works => {
+test("should return path to binary and work", () => {
```

Comment:
can you please use `node:assert`'s methods, single quotes and un-prettify the changes?

Reply:
Sorry, did not see that the formatter has messed with that part.